tools/devtool

```diff
@@ -621,27 +653,20 @@ cmd_test() {
     say "Kernel version: $(uname -r)"
     say "$(sed '/^processor.*: 0$/,/^processor.*: 1$/!d; /^processor.*: 1$/d' /proc/cpuinfo)"
     say "RPM microcode_ctl version: $(rpm -q microcode_ctl)"
-    say "Starting test run ..."
 
     # Testing (running Firecracker via the jailer) needs root access,
     # in order to set-up the Firecracker jail (manipulating cgroups, net
     # namespaces, etc).
     # We need to run a privileged container to get that kind of access.
     env |grep -P "^(AWS_EMF_|BUILDKITE|CODECOV_)" > env.list
+    if [[ "$BUILDKITE" == "true" ]] && [[ "$(uname --machine)" == "x86_64" ]]; then
+      say "Detected CI and performance tests, tuning CPU frequency scaling and idle states for reduced variability"
```

Comment:
I remember there was a problem with this condition, did you find out what it was?

Reply:
Ahhh, right, forgot about that, thanks!

Seems that now its even broken in the PR CI, let me investigate what's happening here :thinking:

Reply:
seems that for some reason the "BUILDKITE" environment variable is unset inside of `cmd_test` when running things in my private pipeline.Not quite sure why, because I dump `env | BUILDKITE` right before running devtool and that contains `BUILDKITE=true`

Reply:
This was because in my own pipeline I was running devtool as `sudo`, which reset environment variables. So its not a problem with the condition itself, just a problem with me haha